src/Mindee/Input/Compressor.cs

```diff
@@ -0,0 +1,282 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Docnet.Core;
+using Docnet.Core.Models;
+using Docnet.Core.Readers;
+using Mindee.Exceptions;
+using SkiaSharp;
+
+namespace Mindee.Input
+{
+    /// <summary>
+    /// Compressor static class to handle image and PDF compression.
+    /// </summary>
+    public static class Compressor
+    {
```

Comment:
Not necessarily on input, I would see this as generic Mindee image utilities.

Was not very clear first time around ...

Maybe a `Mindee.Image` namespace, like we have for Pdf?

Reply:
It's technically format agnostic, I'd rather keep it as an "input operation"

Reply:
Or I can move the image exclusive operations to the Image namespace, and the PDF exclusive operations to the Pdf namespace